### ISSUE TITLE
lockfile issues

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#630]
Hi all
for the Future Teacher project a few of us have regularly been working on a shared LO and although I've known intermittent issues with lockfile previously I've usually blamed others ( like Alistair ;-) ) as I've not previously known others to say that an LO is saying that I'm still editing when I'm not. However maybe because we've been doing this over a regular and extended period 3 of us have all experienced occasions when we've gone to edit, seen the lockfile message that someone else is editing, checked with them by email or phone and discovered that they aren't editing. I haven't been able to identify when and why this happens but we also have all noticed that sometimes when logging out we see a warning message that editor windows are still open when they aren't. That might be coincidence or might not. So a few questions:

1. Has anyone else had issues with lockfile showing that someone is editing when they aren't?
2. Has anyone else had that editor window warning when there isn't an editor window open.
3. How best to troubleshoot the lockfile functionality to identify when/why this happens?
Ron

**Comment by FayCross** [comment]
1 & 2 Yes (but not been able to work out why)

**Comment by JohnSmith-LT** [comment]
Yes, i've experienced that a few times here too. We have a module where student groups develop Xerte LOs and a couple of times i've had to manually remove the lock file...

**Comment by FayCross** [comment]
I've just opened a new issue relating to this (perhaps old lockfiles can be ignored after a certain amount of time) - https://github.com/thexerteproject/xerteonlinetoolkits/issues/1085
